build.gradle.kts

```diff
@@ -57,15 +47,20 @@ dependencies {
   runtimeOnly("org.postgresql:postgresql:42.3.1")
 }
 
+java {
+  toolchain.languageVersion.set(JavaLanguageVersion.of(17))
+}
+
 tasks {
   test {
     useJUnitPlatform()
     exclude("**/integration/*")
   }
 
-  withType<JavaCompile> {
-    sourceCompatibility = JavaVersion.VERSION_11.toString()
-    targetCompatibility = JavaVersion.VERSION_11.toString()
+  withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
+    kotlinOptions {
+      jvmTarget = "17"
```

Comment:
Now that we have the latest HMPPS spring boot plugin and Kotlin plugin we can finally replace Java 11 with 17.